docs/setup-cluster/deploy-cluster/slurm/hpc-with-agent.rst

```diff
@@ -0,0 +1,153 @@
+.. _hpc-with-agent:
+
+####################
+ Agent on Slurm/PBS
+####################
+
+As an alternative to using the HPC Launcher, you may instead utilize the Determined agent. In this
+usage model, the system administrator creates a custom resource pool for each Determined user. You
```

Comment:
1. So the per-user resource pool has no association with any nodes.  It's just a name for the user to track his resource usage when jobs are submitted?
2. The association between the per-user resource pool and the nodes happens when the `user` manually starts the agent on a set of nodes with a configuration file that contains their resource pool name?  What if the user forgets to include their resource pool name in their config file?  Or, what happens if the user specifies another user's resource pool name?

Reply:
1. yes
2. Yes. The default resource pool name is 'default'.  If it exists, it will put the resources there, otherwise the agent will crash.   If they use the wrong resource pool the resources will be added to it.